modcc/symge.hpp

```diff
@@ -46,6 +46,8 @@ struct symbol_term {
     symbol left, right;
 
     symbol_term() = default;
+    symbol_term(const symbol left): left(std::move(left)) {}
```

Comment:
```suggestion
    symbol_term(symbol left): left(std::move(left)) {}
```

Reply:
oh yes, I must have missed to delete those.